tests/wicked/before_test.pm

```diff
@@ -67,6 +67,9 @@ sub run {
         file_content_replace('/etc/sysconfig/dhcpd', '--sed-modifier' => 'g', '^DHCPD_INTERFACE=.*' => 'DHCPD_INTERFACE="' . $ctx->iface() . '"');
         systemctl 'enable --now dhcpd.service';
         if (check_var('WICKED', 'ipv6')) {
+            $self->get_from_data('wicked/dhcp/dhcpd4.conf', '/etc/dhcpd.conf');
```

Comment:
I know that it looks better like this from code perspective , but it also looks odd from test flow perspective : 
1. we create one dhcpd config 
2. modify values there 
3. start DHCP server 
4. overwrite dhcpd.conf with another setup 
5. restart DHCP server 

Also I remember that we have discuss this and agreed that we will change IP ranges **after** refactoring of our test infra , just note to myself let's discuss this on Monday

Reply:
yeah, the reason why I needed a new dhcp config is cause it need to provide 10.20.30.X range to differenciate between dynamic and static address. But we can remove this and rely on getting same IP range for static and dynamic and change it later in the refactor.